### PR TITLE
WT-13405 Bring the Catch2 unit test code in line with the WiredTiger C++ style

### PR DESCRIPTION
**Pull request by luke-pearson**
This change is a style change only. There is a follow up ticket to make the test themselves more consistent which will be more work that what this was.

### CODE REVIEW

File: test/unittest/tests/block/test_block_ckpt.cpp
Comment (mm-ng) [inline test/unittest/tests/block/test_block_ckpt.cpp:53]: Can we change `expectedBits` to the correct case?

File: test/unittest/tests/block/test_bitstring.cpp
Comment (mm-ng) [inline test/unittest/tests/block/test_bitstring.cpp:13]: Can you please fix the `bitVectorSize` and `bitVector` variables too?

**Review by jiechenbo — APPROVED** [review]
LGTM! after Monica's changes.

File: test/unittest/tests/block/test_bitstring.cpp
Comment (luke-pearson) [reply test/unittest/tests/block/test_bitstring.cpp:13]: ac72d1a85

File: test/unittest/tests/block/test_block_ckpt.cpp
Comment (luke-pearson) [reply test/unittest/tests/block/test_block_ckpt.cpp]: ac72d1a85

**Comment by luke-pearson** [comment]
I've made the suggested changes from Monica and resolved merge conflicts due to Jie's recent PR going in. Could we merge this before we merge anything else to minimize further merge conflicts?

**Review by mm-ng — APPROVED** [review]
LGTM

**Pull request merged**
